modules/administration/pages/auth-methods-sso-example.adoc

```diff
@@ -60,9 +61,8 @@ docker run -td --name=idp -p 8080:8080 -e KEYCLOAK_USER=<user> -e KEYCLOAK_PASSW
 * In the ``Name`` field, enter a name for the realm.
     For example, ``SUMA``.
 * Toggle the ``Enabled`` switch to ``On``.
-* In the ``Endpoints`` field, type ``SAML 2.0 Identity Provider Metadata``.
-    This endpoint is ``<IdP_Address>/auth/realms/<Realm_Name>/protocol/saml/descriptor`` which describes the endpoints and certificate in the {productname} configuration file.
-// Probably needs more explanation here. --LKB 20200415
+* In the ``Endpoints`` field, click on the link ``SAML 2.0 Identity Provider Metadata``.
+    This will lead you to ``<IdP_URL>/auth/realms/<Realm_Name>/protocol/saml/descriptor``, where you will see the endpoints and certificate to copy into the {productname} configuration file.
```

Comment:
```suggestion
* In the ``Endpoints`` field, click the ``SAML 2.0 Identity Provider Metadata`` link.
  This will lead you to ``<IdP_URL>/auth/realms/<Realm_Name>/protocol/saml/descriptor``, where you will see the endpoints and certificate to copy into the {productname} configuration file.
```